src/iframe-attributes.js

```diff
@@ -51,8 +51,7 @@ export function getContextMetadata(
 
   // TODO(alanorozco): Redesign data structure so that fields not exposed by
   // AmpContext are not part of this object.
-  const layoutRect = element.getPageLayoutBox ?
```

Comment:
Yes I am aware that is an AMP element before. Asking because I see https://github.com/ampproject/amphtml/blob/master/src/3p-frame.js#L307